### PR TITLE
fix(condo): DOMA-7763 don't show organization create modal if active invites exists

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

File: apps/condo/domains/organization/components/OrganizationSelect.tsx [hunk 1]
```diff
@@ -32,28 +32,38 @@ function compareEmployees (lhs: OrganizationEmployeeType, rhs: OrganizationEmplo
...
+    }, { skip: isNull(userId) })
+
+    const { count: hasInvites, loading: isInvitesLoading } = OrganizationEmployee.useCount({
+        where: {
+            user: { id: userId },
+            isAccepted: false,
+            ...ORGANIZATION_EMPLOYEE_WHERE_QUERY,
         },
-    })
+    }, { skip: isNull(userId) })
```
Comment (SavelevMatthew) [inline apps/condo/domains/organization/components/OrganizationSelect.tsx:66]: Isn't skip is `Int`?

Comment (SavelevMatthew) [reply apps/condo/domains/organization/components/OrganizationSelect.tsx:66]: Why not just `!userId` :)

Comment (sitozzz) [reply apps/condo/domains/organization/components/OrganizationSelect.tsx:66]: If I'm not mistaken, I used an apollo graphql client hook parameter. According to the documentation it has a boolean type
image

Comment (sitozzz) [reply apps/condo/domains/organization/components/OrganizationSelect.tsx:66]: > Why not just `!userId`? :)

Yup, it is possible

**Review by Alllex202 — COMMENTED** [review]
Will modal be shown again if user declines all invitations?

**Review by Alllex202 — APPROVED** [review]

**Comment by sitozzz** [comment]
> Will modal be shown again if user declines all invitations?

No it didn't show again. Only if user declines all of his invitations and after that reload the page

**Pull request merged**